src/tests/fs_tests.rs

```diff
@@ -3047,6 +3047,8 @@ pub mod fs_tests {
 
         let cage = interface::cagetable_getref(1);
         let path = "/parentdir/dir";
+         // Ensure the directories do not exist for clean environment setup
+        let _ = Cage::rmdir_recursive_syscall(&cage, "/parentdir");
```

Comment:
where do we define this syscall?

Reply:
Sorry, I was in process of making changes and pushed a commit. Changed now